src/content/overview/developer-portal/_index.md

```diff
@@ -1,12 +1,33 @@
 ---
 title: Developer portal
-description: The central Giant Swarm self-service portal for your platform engineers and developers.
+description: Our Backstage-based developer portal is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so engineers using your platform find all the information they need in the right place.
 weight: 30
 menu:
   principal:
     parent: overview
     identifier: overview-developer-portal
-last_review_date: 2024-03-18
+last_review_date: 2024-07-01
 owner:
-  - https://github.com/orgs/giantswarm/teams/sig-product
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
 ---
+
+Our developer portal absed on [Backstage](https://www.cncf.io/projects/backstage/) is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.
+
+Giant Swarm can assist you in various ways depending on where you are in your journey towards your own developer portal. We provide a pre-configured Backstage instance as a starting point to customers who don't run their own instance yet. We also provide plugins that can be used to extend the functionality of your Backstage instance.
+
+## Why developer portals
+
+The Giant Swarm platform combines capabilities provided by a variety of open source tools, and likely your engineers are using even more tools to accomplish their goals. Developer portals should serve as a single entry point and provide a navigation path to the tools engineers need access to. Furthermore, they can give an overview of the software landscape and the platform itself, make documentation and troubleshooting runbooks accessible, and assist in the creation of new services according to platform standards.
```

Comment:
This doesn't say much about which capabilities. Can we link to the platform overview, or give examples here, without making this too long? Developer portals are a topic where many companies don't see the benefit, so examples are critical.

And let's add `runbooks` to the allowlist of vale.

Reply:
> Developer portals are a topic where many companies don't see the benefit, so examples are critical.

I don't second this. Also, we'll add more details once we can.